site/content/flowcharts/lonecoder.md

```diff
@@ -24,5 +24,62 @@ graph TD;
   class Link,B,C,F linkedBox
 {{< /mermaid >}}
 
-To read about this workflow in detail, please see
-[single developer process](/dev-review/recipes/lonecoder)
+### Find a reviewer
+
+There is no code review without (a) reviewer(s). The good news is,
+almost any researcher who code is a good candidate for being
+one. 
+
+Potential reviewers are not far: in your own research group, others
+you collaborate with or your local Research Software Engineering
+group. They can also be found in communities outside your institution
+or specific research domain. In the [Find a reviewer](https://researchcodereviewcommunity.github.io/dev-review/recipes/find_a_reviewer/) phase, we look at
+different pools of potential reviewers, what makes a suitable
+reviewer, and ways of getting in touch.
+
+### Meet and agree on objectives
```

Comment:
I was of the opinion that this title might need to change. Something like "Pre-review (or Preliminary?) Meeting" which encompasses "Meet and agree on objectives" and the other points from our detailed flowchart like "Explain scientific context" and "Decide on section of code to review".

However, I am quite happy to leave it as this because is gives the main thrust of what this initial meeting should be, which is to set the objectives.